### PR TITLE
Revamp Quickstart

### PR DESCRIPTION
**Pull request by tbedford**
## Description

* Adds new Quickstart
* Adds Quix Tour
* Reorganized definitions into glossary
* Adds "push data using Quix Streams" section to "How to connect to Quix"
* Makes "How to connect to Quix" more prominent
* **Rename "How to connect to Quix" to Ingest data**
* Removes old Quickstart (but preserves polling code as How-To)
* Updates polling code to use `users` endpoint, rather than `beers` endpoint
* Adds CSV ingest topic to How-Tos
* Adds CSV ingest section to "Connect to Quix" (links to How-To)

## Review

* Best start at the Quickstart. 
* Step through the Quix Tour.
* Check the items in the description.

[Page to review](https://quixdocsdev.blob.core.windows.net/pr174/index.html)

### CODE REVIEW

File: docs/platform/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+pip install python-dotenv
+```
+
+!!! tip
+
+    If you're on Mac and using Homebrew, you may have multiple Python versions installed. In this case you may have to use the command `pip3` to install your modules. 
+
+You're going to use the [Quix Streams](../client-library-intro.md) library to push data into Quix Platform. This is just one of [many ways](../platform/ingest-data.md) to get your data into Quix. You could for example simply log into Quix and use one of our already available [connectors](../platform/connectors/index.md), but where's the fun in that!
+
+You use the `psutil` module to retrieve the CPU load on your laptop.
```
Comment (SteveRosam) [inline docs/platform/quickstart.md:97]: Should these be You'll or you will? Up to you. This is just my suggestion.

File: docs/platform/quickstart.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+## 4. Run your code
+
+Run your code with the following command in your terminal:
+
+```
+python cpu_load.py
+```
+
+The code runs and displays your CPU load. The code is now publishing data to the Quix topic `cpu-load`.
```
Comment (SteveRosam) [inline docs/platform/quickstart.md]: Maybe `The code runs and, after creating the cpu-load topic, displays your CPU load.`

File: docs/platform/quickstart.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+
+## 5. See the data in Quix
+
+1. Switch back to the Quix Portal.
+2. Click on `Topics` in the main left-hand navigation.
+3. You see the `cpu-load` topic. Note the vertical green bars representing inbound data.
+4. Hover the mouse over the `Data` colum. You see the tool tip text `View live data`.
+5. Click the mouse where the tool tip text is displayed. You are taken to the Quix Data Explorer in a new tab.
+6. Under `SELECT STREAMS` select the box `Quickstart CPU Load - Server 1`.
+7. Under `SELECT PARAMETERS OR EVENTS` select `CPU_Load_1`. Your real-time CPU load is displayed as a waveform. You can also take a look at the table view, and the message view.
```
Comment (SteveRosam) [inline docs/platform/quickstart.md]: Mine just says `CPU_Load` without the 1

File: docs/platform/quixtour/process-threshold.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# Process - threshold detection
+
+In this part of the tour you'll learn how to create a transform. The transform detects if CPU load exceeds a certain threshold, and of so, send a dataframe to its output topic.
```
Comment (SteveRosam) [inline docs/platform/quixtour/process-threshold.md]: and of so, send a dataframe  'sends'?

File: docs/platform/quixtour/process-threshold.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+## Watch the video
+
+<div style="position: relative; padding-bottom: 62.5%; height: 0;"><iframe src="https://www.loom.com/embed/14daea2ac1a044cd84cf47ce1fe37c03?sid=4e72a0f8-8276-408b-beaf-1515454d8c49" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+## Create the transform
+
+To create the threshold detection transform:
+
+1. Click on `Code Samples` in the main left-hand navigation. 
+2. Select the `Python`, `Transformation`, and `Basic templates` facets.
```
Comment (SteveRosam) [inline docs/platform/quixtour/process-threshold.md]: we need to align on terminology here. I've called the filters. I dont mind facets though. But what do the rest or the org call them?

File: docs/platform/quixtour/serve-sms.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+    def on_stream_received_handler(stream_consumer: qx.StreamConsumer):
+        stream_consumer.timeseries.on_dataframe_received = on_dataframe_received_handler
+
+    topic_consumer.on_stream_received = on_stream_received_handler
+    print("Listening to streams. Press CTRL-C to exit.")
+    qx.App.run()
+    ```
+
+11. Click the add file icon to add a new file to your project - name it `.env`.
```
Comment (SteveRosam) [inline docs/platform/quixtour/serve-sms.md:79]: Quix Platform already supports environment variables. Do we need to guide them to create .env files for this?

**Comment by SteveRosam** [comment]
I couldnt find  "How to connect to Quix"

**Review by SteveRosam — DISMISSED** [review]

**Comment by tbedford** [comment]
> I couldnt find "How to connect to Quix"

That has been renamed to:

https://quixdocsdev.blob.core.windows.net/pr174/platform/ingest-data.html

Ah, yes, the description of the PR wasn't updated, sorry about that! I've now added the rename to the list of items in the description.

File: docs/platform/quickstart.md (hunk 1 above)
Comment (tbedford) [reply docs/platform/quickstart.md:97]: Writing style guide is present tense, but I know what you mean.

File: docs/platform/quickstart.md (hunk 3 above)
Comment (tbedford) [reply docs/platform/quickstart.md]: Well spotted.

File: docs/platform/quixtour/process-threshold.md (hunk 5 above)
Comment (tbedford) [reply docs/platform/quixtour/process-threshold.md]: Let's standardize on filters. I will fix.

File: docs/platform/quixtour/serve-sms.md (hunk 6 above)
Comment (tbedford) [reply docs/platform/quixtour/serve-sms.md:79]: Good point. I will take a look at see if it's a better approach.

File: docs/platform/quixtour/process-threshold.md (hunk 4 above)
Comment (tbedford) [reply docs/platform/quixtour/process-threshold.md]: oops.

File: docs/platform/quickstart.md (hunk 2 above)
Comment (tbedford) [reply docs/platform/quickstart.md]: I will reword.

File: docs/platform/quixtour/serve-sms.md (hunk 6 above)
Comment (tbedford) [reply docs/platform/quixtour/serve-sms.md:79]: Yes, this is a better way. Going forwards we should do this. I've added a tip in the section on how to do this approach. I've also added a new "How-To" on how to do this.

**Review by SteveRosam — DISMISSED** [review]

**Comment by tbedford** [comment]
I've put to blocked as I now have a serious issue with videos hosted on Loom.

**Review by SteveRosam — DISMISSED** [review]

**Review by SteveRosam — APPROVED** [review]

**Pull request merged**